packages/browser/src/Component/Widget/Server/Document/Link/View/DocumentLinkInterface.js

```diff
@@ -4,7 +4,11 @@ import { DocumentProvider } from '../../Core/ViewModel/DocumentProvider';
 import { Link } from '../Model/Link';
 import { LinkProvider } from '../ViewModel/LinkProvider';
 
-import { findChildByID } from '../../../../../HTMLUtil';
+import {
+  createDisplayable,
+  createLabelInput,
+  findChildByID,
+} from '../../../../../HTMLUtil';
```

Comment:
For my knowledge (and improve my JS competence) : since HTMLUtils is exported in index.js, is it possible to make the import from it ? 
This means that if we move HTMLUtil one day, we will only have to change the path in index.js (+ this can improve readability)

Reply:
Yes I was thinking about that, one solution is what you say but we are loosing some information about dependencies between "modules"/subfolder since every file (theoratically) depends of index.js (aka @ud-viz/browser).
I was thinking about another way a bit more complex but allowing a good readability about dependencies which is each file can import module from the first api file up (for this file it would be [this one](https://github.com/VCityTeam/UD-Viz/blob/master/packages/browser/src/Component/Widget/Server/Document/Document.js)) and so on.

Not sure to be clear but we could create an issue about it